src/main/java/uk/gov/hmcts/ccd/domain/model/aggregated/CaseViewFieldBuilder.java

```diff
@@ -23,43 +23,43 @@ public CaseViewFieldBuilder(final CompoundFieldOrderService compoundFieldOrderSe
         this.compoundFieldOrderService = compoundFieldOrderService;
     }
 
-    public CaseViewField build(CaseField caseField, CaseEventField eventField) {
+    public CaseViewField build(CaseFieldDefinition caseFieldDefinition, CaseEventFieldDefinition eventField) {
```

Comment:
we should rename the parameter too `eventField` to `eventFieldDefinition`

Reply:
renamed